ansible/roles/products/common/clear-icons/tasks/main.yml

```diff
@@ -24,9 +24,13 @@
   find:
     paths: "{{ desktop_path }}"
     patterns: '*.desktop'
-    contains: "^.*Comment=autogenerated_aurora"
+    contains: "^.*Comment=autogenerated_aurora_{{ container_name }}"
   register: file_list
 
+- name: debug
+  debug:
+    msg: "{{ file_list }}"
+
 - name: Delete all .desktop files with Shadow in the comment
```

Comment:
with autogenerated_aurora_{{ container_name }}

Reply:
What do you mean?

Reply:
The name of the task in line 34 should be different (not really important)